plugins/module_utils/template.py

```diff
@@ -440,3 +440,17 @@ def get_route_map_policy_for_multicast_uuid(self, route_map_policy_for_multicast
         kv_list = [KVPair("name", route_map_policy_for_multicast_name)]
         match = self.get_object_by_key_value_pairs("Route Map Policy for Multicast", existing_route_map_policies, kv_list, fail_module=True)
         return match.details.get("uuid")
+
+
+class MSOTemplateCache:
```

Comment:
If it possible to move self.cache = {} to init function or constructor of class MSOTemplate itself?

Reply:
Done